content/rs/references/cli-utilities/rladmin/upgrade.md

```diff
@@ -0,0 +1,107 @@
+
+For more information, see [Upgrade an existing Redis Software Deployment]({{< relref "/rs/installing-upgrading/upgrading" >}}).
+
+```sh
+rladmin upgrade db { db:<id> | <name> }
+                [ preserve_roles ]
+                [ keep_redis_version ]
+                [ discard_data ]
+                [ force_discard ]
+                [ parallel_shards_upgrade ]
+                [ keep_crdt_protocol_version ]
+                [ redis_version <version> ]
+                [ force ]
+                [ { latest_with_modules | and module module_name <mod_name> version <version> module_args <args_str> } ]
+```
+
+### Parameters
+
+| Parameters                 | Type/Value               | Description                                                                                                            |
+|----------------------------|--------------------------|------------------------------------------------------------------------------------------------------------------------|
+| db                         | db:\<id\> <br />name     | Database to upgrade                                                                                                    |
+| and module | [upgrade module](#upgrade-module) command | Clause that allows upgrade of database and specified Redis module in a single step with only one restart (can be specified multiple times)  |
+| discard_data               |                          | Indicates that data will not be saved after the upgrade                                                                |
+| force                      |                          | Forces upgrade and skips warnings and confirmations                                                                    |
+| force_discard              |                          | Forces `discard_data` if there is replication or persistence enabled                                                   |
+| keep_crdt_protocol_version |                          | Keeps the current crdt protocol version                                                                                |
+| keep_current_version       |                          | Upgrades to a new patch release, not to the latest major.minor version                                                 |
+| latest_with_modules        |                          | Upgrades both the database and all modules in the database                                                             |
+| parallel_shards_upgrade    | Integer <br />all        | Maximum number of shards to upgrade all at once                                                                        |
+| preserve_roles             |                          | Performs an additional failover to guarantee roles of shards are preserved                                             |
+| redis_version              | Redis version            | Upgrades database to the specified version instead of the latest version                                               |
```

Comment:
```suggestion
| redis_version              | Redis version            | Upgrades the database to the specified version instead of the latest version                                               |
```